src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -518,7 +548,11 @@ public void Initialize()
 
                 if (ScriptConfig.FileLoggingMode != FileLoggingMode.Never)
                 {
-                    PurgeOldLogDirectories();
+                    using (metricsLogger.LatencyEvent(MetricEventNames.HostStartupPurgeLogDirectoriesLatency))
+                    {
+                        PurgeOldLogDirectories();
+                        TraceWriter.Verbose("Old log directories purged.");
```

Comment:
PurgeOldLogDirectories is already logging. If we need more logs, perhaps add the log in that method?

Reply:
It currently doesn't log if nothing was deleted, so we want something here because it takes time to even scan for stuff to delete. The function has a short circuit return so I'll leave the verbose logging statement here for now.